src/test/resources/user-service/user.json

```diff
@@ -0,0 +1,39 @@
+{
+  "username": "jllopes",
+  "website": "",
+  "display_name": "João Lopes",
+  "account_id": "123abc456def789ghi101jkl",
+  "links": {
+    "hooks": {
+      "href": "https:\/\/bitbucket.org\/!api\/2.0\/users\/jllopes\/hooks"
+    },
+    "self": {
+      "href": "https:\/\/bitbucket.org\/!api\/2.0\/users\/jllopes"
+    },
+    "repositories": {
+      "href": "https:\/\/bitbucket.org\/!api\/2.0\/repositories\/jllopes"
```

Comment:
I don't think that this hrefs come escaped like this, can you do a raw curl or something so we can avoid any escaping on top?

Reply:
This is a direct copy-paste from the existing tests, we're also not testing these values.